Description: Fix geany profile
 Geany is the default lightweight ide of the Parrot Project.
 We need it to be more permissive for general purpose
 programming and to edit system files and configuratons.
Author: Lorenzo "Palinuro" Faletra <palinuro@parrotsec.org>
Last-Update: 2018-05-25

--- firejail-0.9.54.orig/etc/geany.profile
+++ firejail-0.9.54/etc/geany.profile
@@ -7,7 +7,7 @@ include /etc/firejail/globals.local
 
 noblacklist ${HOME}/.config/geany
 
-include /etc/firejail/disable-common.inc
+# include /etc/firejail/disable-common.inc
 include /etc/firejail/disable-passwdmgr.inc
 include /etc/firejail/disable-programs.inc
 
@@ -16,8 +16,8 @@ netfilter
 no3d
 nodvd
 nogroups
-nonewprivs
-noroot
+# nonewprivs
+# noroot
 nosound
 notv
 novideo
@@ -26,4 +26,4 @@ seccomp
 shell none
 
 private-dev
-private-tmp
+# private-tmp
